server/routes/appointments/feedback/initialAssessment/checkYourAnswers/initialAssessmentFeedbackCheckAnswersPresenter.ts

```diff
@@ -1,23 +1,24 @@
 import DeliusServiceUser from '../../../../../models/delius/deliusServiceUser'
 import CheckFeedbackAnswersPresenter from '../../shared/checkYourAnswers/checkFeedbackAnswersPresenter'
-import AttendanceFeedbackPresenter from '../../shared/attendance/attendanceFeedbackPresenter'
-import InitialAssessmentAttendanceFeedbackPresenter from '../attendance/initialAssessmentAttendanceFeedbackPresenter'
 import { InitialAssessmentAppointment } from '../../../../../models/appointment'
-import InitialAssessmentBehaviourFeedbackPresenter from '../behaviour/initialAssessmentBehaviourFeedbackPresenter'
+import FeedbackAnswersPresenter from '../../shared/viewFeedback/feedbackAnswersPresenter'
+import AttendanceFeedbackQuestionnaire from '../../shared/attendance/attendanceFeedbackQuestionnaire'
+import BehaviourFeedbackQuestionnaire from '../../shared/behaviour/behaviourFeedbackQuestionnaire'
 
 export default class InitialAssessmentFeedbackCheckAnswersPresenter extends CheckFeedbackAnswersPresenter {
-  protected readonly attendancePresenter: AttendanceFeedbackPresenter
-
-  protected readonly behaviourPresenter: InitialAssessmentBehaviourFeedbackPresenter
+  readonly feedbackAnswersPresenter: FeedbackAnswersPresenter
 
   constructor(
     appointment: InitialAssessmentAppointment,
     private readonly serviceUser: DeliusServiceUser,
     private readonly referralId: string
   ) {
     super(appointment)
-    this.attendancePresenter = new InitialAssessmentAttendanceFeedbackPresenter(appointment, this.serviceUser)
-    this.behaviourPresenter = new InitialAssessmentBehaviourFeedbackPresenter(appointment, this.serviceUser)
+    this.feedbackAnswersPresenter = new FeedbackAnswersPresenter(
+      appointment,
+      new AttendanceFeedbackQuestionnaire(appointment, serviceUser),
+      new BehaviourFeedbackQuestionnaire(appointment, this.serviceUser)
```

Comment:
nit: is there a reason for `this.serviceUser` vs `serviceUser` above?

Reply:
They're the same thing in this case, but I've changed it to be more consistent.

Reply:
Yeah - it's weird, my editor sometimes marks it as an usused variable if I do one but not the other, I don't know which. Feels good to be consistent though!